packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -657,19 +693,24 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 		if (newDevice.initialized === true) {
 			throw new Error('Device ' + deviceId + ' is already initialized!')
 		}
-
 		this.emit(
 			'info',
 			`Initializing device ${newDevice.deviceId} (${newDevice.instanceId}) of type ${DeviceType[deviceOptions.type]}...`
 		)
-
-		await newDevice.init(deviceOptions.options, activeRundownPlaylistId)
-
-		await newDevice.reloadProps() // because the device name might have changed after init
-
-		this.emit('info', `Device ${newDevice.deviceId} (${newDevice.instanceId}) initialized!`)
-
-		return newDevice
+		return makeAbortable(async () => {
+			const throwIfAborted = () => {
+				if (options?.signal?.aborted) {
```

Comment:
This if-statement is identical to the one on line 684. Would be nice to reduce duplication